scene/gui/popup_menu.cpp

```diff
@@ -876,6 +893,86 @@ void PopupMenu::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_INTERNAL_PROCESS: {
+			Input *input = Input::get_singleton();
+
+			if (input->is_action_just_released("ui_left") || input->is_action_just_released("ui_right") || input->is_action_just_released("ui_up") || input->is_action_just_released("ui_down")) {
```

Comment:
This stops movement if unrelated action is released. Not sure if intended.

Reply:
I removed ui_left and ui_right